pytorch_lightning/profiler/pytorch.py

```diff
@@ -195,7 +195,6 @@ def __call__(self, num_step: int) -> "ProfilerAction":
 class PyTorchProfiler(BaseProfiler):
```

Comment:
There are still some mentions in the docs of
pytorch_lightning/profiler/__init__.py

Reply:
thanks, fixed!